sdk/state/close.go

```diff
@@ -93,7 +93,7 @@ func (c *Channel) ProposeClose() (CloseAgreement, error) {
 	}
 
 	// If the channel is not open yet, error.
-	if c.latestAuthorizedCloseAgreement.isEmpty() {
+	if c.latestAuthorizedCloseAgreement.isEmpty() || !c.openExecutedAndValidated {
```

Comment:
grouped these checks together, I think if either fail we'd want to throw the same error